cheroot/test/test_more.py

```diff
@@ -0,0 +1,57 @@
+import os
+import pytest
+import requests
+import ssl
+from cheroot.ssl.builtin import BuiltinSSLAdapter
+
+def ssl_file(filename):
```

Comment:
Can you use `trustme` for SSL certificates creation/testing?

Reply:
I think that is relatively straightforward to do.

Reply:
Yeah, please do